src/web/app/components/question-submission-form/question-submission-form.component.ts

```diff
@@ -94,10 +94,29 @@ export class QuestionSubmissionFormComponent implements OnInit {
               private feedbackResponseService: FeedbackResponsesService) {
     this.visibilityStateMachine =
         this.feedbackQuestionsService.getNewVisibilityStateMachine(
-            this.model.giverType, this.model.recipientType);
+          this.model.giverType, this.model.recipientType);
   }
 
   ngOnInit(): void {
+    // Sorts the receipients alphabetically
+    this.model.recipientSubmissionForms.sort((firstRecepient: FeedbackResponseRecipientSubmissionFormModel,
+      secondRecepient: FeedbackResponseRecipientSubmissionFormModel) => {
+      const indexOne: number = this.model.recipientList.findIndex((x: FeedbackResponseRecipient) =>
+        x.recipientIdentifier === firstRecepient.recipientIdentifier);
+
+      const indexTwo: number = this.model.recipientList.findIndex((x: FeedbackResponseRecipient) =>
+        x.recipientIdentifier === secondRecepient.recipientIdentifier);
+
+      if (indexOne < indexTwo) {
+        return -1;
+      }
+
+      if (indexOne > indexTwo) {
+        return 1;
+      }
+
+      return 0;
+    });
```

Comment:
Perhaps this can be simplified into `return indexOne - indexTwo` since they are both numbers.